server/server/.env.sample

```diff
@@ -0,0 +1,9 @@
+DATABASE_NAME=dbname
+DATABASE_USER=admin
+DATABASE_PASSWORD=secret-password
+DATABASE_HOST=localhost
+DATABASE_PORT=5432
```

Comment:
If we don't need these, remove. If we do and they should be same as line :6, please fix.